--- libdc1394-1.2.1/configure.in.bero	2013-01-15 23:22:08.516674026 +0100
+++ libdc1394-1.2.1/configure.in	2013-01-15 23:22:22.667505065 +0100
@@ -1,8 +1,8 @@
 # process this file with autoconf to get a configure script
-AC_INIT(libdc1394/dc1394_control.c)
+AC_INIT(libdc1394, 1.2.1)
 
-AM_INIT_AUTOMAKE(libdc1394, 1.2.1)
-AM_CONFIG_HEADER(config.h)
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
 
 AC_PROG_CC
 AC_PROG_LIBTOOL
